ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -34,12 +34,28 @@ using namespace llvm::symbolize;
 using namespace std;
 
 // The bitcode module loaded from the .llvmbc section of the currently-running
-// binary. This is shared across threads.
+// binary. This cannot be shared across threads and used concurrently without
+// acquiring a lock, and since we do want to allow parallel compilation, each
+// thread takes a copy of this module.
 ThreadSafeModule GlobalAOTMod;
 
-// Flag used to ensure that bitcode is loaded only once.
+// Flag used to ensure that GlobalAOTMod is loaded only once.
 once_flag GlobalAOTModLoaded;
 
+// The thread's copy of the GlobalAOTMod.
```

Comment:
s/The thread's/A thread local/ ?

Reply:
It should be obvious that it's a thread local from `thread_local`. Crucially, it's a copy of a global variable, which is subtly different.

Reply:
The comment doesn't parse to me at the moment, hence my suggestion. I'm prepared to admit that my suggestion might not be perfect, but I think we need to do do something to clarify matters.

Reply:
I'm confused. It parses fine, doesn't it?

A few lines above we have a comment which explains that each thread needs a copy of the `GlobalAOTMod`.

Then here:
> The thread's copy of the GlobalAOTMod.

i.e. The copy of the GlobalAOTMod as owned by the thread.

Call me out if I'm being silly :P

Reply:
It doesn't parse to me.

Reply:
Is f5deac clearer?

Reply:
Yes, it's a bit clearer, but it still seems less clear than my suggestion. However, I don't want to fight on this one, so let's go with it.